### PR TITLE
Test globs in module specs (RhBug:1644588)

### PR DESCRIPTION
**Pull request by dmach**

### CODE REVIEW

File: dnf-behave-tests/features/module-install-globs.feature [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+        | Action                    | Package                                           |
+        | module-stream-enable      | nodejs:10                                         |
+        | module-profile-install    | nodejs/default                                    |
+        | install                   | nodejs-1:10.11.0-1.module_2200+adbac02b.x86_64    |
+
+
+Scenario: Install a module profile with glob in version
+  Given I use the repository "dnf-ci-fedora-modular-updates"
+   When I execute dnf with args "module install nodejs:10:20180920*"
+   # by selecting version, the older RPM from dnf-ci-fedora-modular gets installed
```
Comment (lukash) [inline dnf-behave-tests/features/module-install-globs.feature]: Looks like a copy-paste leftover.

Comment (dmach) [reply dnf-behave-tests/features/module-install-globs.feature]: Could you be more specific?
If you mean the comment, it's there by purpose - to give a hint how the test is performed

Comment (lukash) [reply dnf-behave-tests/features/module-install-globs.feature]: Sorry, I was too fast, misread the comment... I thought the comment is talking about the actual RPM software, but it is talking about an RPM package. Apparently, you're correct in your nomenclature :grin: quoting wikipedia:

> The name RPM refers to the following: the .rpm file format, files in the .rpm file format, software packaged in such files, and the package manager program itself.

Doesn't really help though :slightly_smiling_face:

Comment (dmach) [reply dnf-behave-tests/features/module-install-globs.feature]: Understand. I changed the text slightly to make it less confusing.

**Comment by j-mracek** [comment]
The test looks good. After local testing I will merge it.

**Comment by j-mracek** [comment]
Merged in https://github.com/rpm-software-management/ci-dnf-stack/pull/536

**Pull request closed without merging**